executor/infoschema_reader.go

```diff
@@ -3260,24 +3267,46 @@ func (e *memtableRetriever) setDataFromResourceGroups() error {
 	rows := make([][]types.Datum, 0, len(resourceGroups))
 	for _, group := range resourceGroups {
 		//mode := ""
-		burstable := "NO"
+		burstable := burstdisableStr
 		priority := model.PriorityValueToName(uint64(group.Priority))
-		fillrate := "UNLIMITED"
+		fillrate := unlimitedFillRate
 		isDefaultInReservedSetting := group.Name == "default" && group.RUSettings.RU.Settings.FillRate == math.MaxInt32
 		if !isDefaultInReservedSetting {
 			fillrate = strconv.FormatUint(group.RUSettings.RU.Settings.FillRate, 10)
 		}
+		// convert runaway settings
+		queryLimit := ""
+		if setting := group.RunawaySettings; setting != nil {
+			runawayRule, runawayAction, runawayWatch := "", "", ""
+			if setting.Rule == nil {
+				return errors.Errorf("unexpected runaway config in resource group")
+			}
+			dur := time.Duration(setting.Rule.ExecElapsedTimeMs) * time.Millisecond
+			runawayRule = fmt.Sprintf("%s=%s", "EXEC_ELAPSED", dur.String())
+			runawayAction = fmt.Sprintf("%s=%s", "ACTION", model.RunawayActionType(setting.Action).String())
+			if setting.Watch != nil {
+				dur := time.Duration(setting.Watch.LastingDurationMs) * time.Millisecond
+				runawayWatch = fmt.Sprintf("%s=%s[%s]", "WATCH", model.RunawayWatchType(setting.Watch.Type).String(), dur.String())
+				queryLimit = fmt.Sprintf("%s, %s, %s", runawayRule, runawayAction, runawayWatch)
+			} else {
+				queryLimit = fmt.Sprintf("%s, %s", runawayRule, runawayAction)
+			}
```

Comment:
```suggestion
		// convert runaway settings
		var builder strings.Builder
		if setting := group.RunawaySettings; setting != nil {
			if setting.Rule == nil {
				return errors.Errorf("unexpected runaway config in resource group")
			}
			dur := time.Duration(setting.Rule.ExecElapsedTimeMs) * time.Millisecond
			builder.WriteString("EXEC_ELAPSED=")
			builder.WriteString(dur.String())
			builder.WriteString(", ACTION=")
			builder.WriteString(model.RunawayActionType(setting.Action).String())
			if setting.Watch != nil {
				dur := time.Duration(setting.Watch.LastingDurationMs) * time.Millisecond
				builder.WriteString(", WATCH=")
				builder.WriteString(model.RunawayWatchType(setting.Watch.Type).String())
				builder.Write('[')
				builder.WriteString(dur.String())
				builder.Write(']')
			}
		}
		queryLimit := builder.String()
```
I think use StringBuilder instead of nested fmt.Sprintf can make the code cleaner, but I'm ok with the current code.